mhs/running-mhs-adaptor-locally.md

```diff
@@ -51,4 +51,31 @@ used to facilitate health checks in a load balanced environment.
 Each of these services exposes a `/healthcheck` URL which, when called indicates healthy state by returning an empty HTTP 200 response.
 
 In order to determine the ports on `localhost` which these health-check endpoints are listening on, examine your local copy
-of the [docker-compose](../docker-compose.yml) file.
\ No newline at end of file
+of the [docker-compose](../docker-compose.yml) file.
+
+## Running MHS components and load balancer with docker
+
+To run the MHS services using docker containers follow the steps below:
+
+1. Make a file in the root call `export-env-vars-and-run-mhs-docker.sh` this name must match to be excluded from git
+2. Populate the file with the following information:
+  ```sh
+#!/bin/bash
+echo -e "\033[1;32mExporting environment variables\033[0m"
```

Comment:
Hard to understand at first look
Please add variables as in https://github.com/nhsconnect/integration-adaptors/pull/193/files#diff-b42d9fb95f574b12b6c1a033fa91dddbR3